test/systems/plonk/placeholder/placeholder.cpp

```diff
@@ -266,8 +266,6 @@ ACTOR_THREAD_TEST_CASE(placeholder_permutation_polynomials_test) {
     BOOST_CHECK_MESSAGE(id_res == sigma_res, "Complex check");
 }
 
-/*
```

Comment:
I enabled all the tests that were enabled on the single threaded version. There were 2 tests with *boost::unit_test::disabled(), which I commented out. These 2 are related to lookup argument, which will be updated, no need to fix it.